src/Database/Driver.php

```diff
@@ -258,7 +258,7 @@ public function schemaValue($value)
             is_numeric($value) && strpos($value, ',') === false &&
             $value[0] !== '0' && strpos($value, 'e') === false)
         ) {
-            return $value;
+            return (string)$value;
```

Comment:
It better be :) Because it would be casted right afterwards to a string anyway by PHP when concatinating it into the schema context.